src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -65,8 +68,16 @@
   @JsonProperty
   private ContentEncoding contentEncoding = DEFAULT_CONTENT_ENCODING;
 
+  /**
+   * @deprecated use {@link Builder}
+   */
+  @Deprecated
```

Comment:
Is default constructor for easy deserealisation from json?
If it is so, then it shouldn't be deprecated, but just commented why it exists.